docs/help/git_webhooks.md

```diff
@@ -0,0 +1,78 @@
+## Webhooks
+
+### Give the bot an SSL-certificate
+
+#### Self-signed certificate
+
+***WARNING!*** Your git host may not accept self-signed certificates.
+
+`openssl req -x509 -newkey rsa:4096 -keyout key.pem -out cert.pem -days 3650`
+
+#### [acme.sh](https://github.com/Neilpang/acme.sh) certificate
+
+Acme.sh is a [LetsEncrypt client](https://letsencrypt.org/). If you have
+already [setup it](https://github.com/Neilpang/acme.sh/blob/master/README.md),
+you can install a certificate for bitbot by: `/root/.acme.sh/acme.sh --install-cert -d <domain> --key-file /somewhere/bitbot/owns/key.pem --fullchain-file /somewhere/bitbot/owns/cert.pem`.
+
+***WARNING!*** Always ensure that other users won't be able to read your
+keys. `chmod -R 700 /path/to/directory/containing/<key.pem-and-cert.pem> &&  chown -R bitbotaccount:bitbotgroup /path/to/directory/containing/<key.pem-and-cert.pem>`
+
+### Configure the bot
+
+1. Enable the REST API in bot.conf by adjusting
+
+```
+# key/cert used for REST API
+tls-api-key              = /path/to/key.pem
+tls-api-certificate      = /path/to/cert.pem
+api-port                 = 5000
+```
+
+you may change the API port if you wish.
+
+2. restart the bot or send it a `SIGUSR1` signal
+
+3. enable the REST API by `!config bot rest-api on`
+
+4. start the REST API for the first time by `!reloadmodule rest_api`
+
+5. add your repository by `/msg <bot> apikey <reponame> /api/github`
```

Comment:
Wrong, it's not a repository being added.

```suggestion
5. create a new API key by `/msg <bot> apikey <apikey-name> /api/github`
The `apikey-name` is how you can later revoke it in case of trouble.
```